package.json

```diff
@@ -52,6 +52,8 @@
     "demo": "webpack --config demo/webpack.config.js --mode=development && node demo/data.js && node demo/server.js"
   },
   "jest": {
+    "verbose": true,
+    "testURL": "http://localhost/",
```

Comment:
Tests were failing. After looking into Jest repo/community, this is what the recommendation was.
This might also fix other PRs like #223